app/src/main/java/com/stevesoltys/seedvault/restore/install/ApkRestore.kt

```diff
@@ -55,11 +58,17 @@ internal class ApkRestore(
             installResult[packageName] = ApkInstallResult(
                 packageName = packageName,
                 progress = progress,
-                state = QUEUED,
+                state = if (isAllowedToInstallApks) QUEUED else FAILED,
```

Comment:
So the intention is for this to appear to the user as a failure? Looks like it, and that is what is happening, including for system apps, which is fairly confusing.

<details><summary>Screenshots</summary>

![Screenshot_20240416-063633](https://github.com/seedvault-app/seedvault/assets/7275539/5d5a8e42-b774-4cec-b15e-29bda7227eef)

![Screenshot_20240416-063652](https://github.com/seedvault-app/seedvault/assets/7275539/ba9ecde6-789b-442c-9189-c0f4b3bf7457)

Reply:
This may be the same if apk backup is disabled. I think we show failed there as well whenever we don't have an APK.

Reply:
Yep, this patch tries to just make the policy block same as "we don't have an APK"

Improvements to that definitely welcome separately. We haven't received much feedback about it in general, making me think most users choose to backup APKs.